app/components/ui/atoms/NotificationRibbon.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components'
+import { th } from '@pubsweet/ui-toolkit'
+
+const backgroundColor = ({ type = 'primary', theme }) =>
+  ({
+    error: theme.colorError,
+    success: theme.colorSuccess,
+    primary: theme.colorPrimary,
+  }[type])
+
+export default styled.div`
+  background: ${backgroundColor};
+  color: ${th('colorTextReverse')};
+  padding: ${th('space.2')};
+  font-family: ${th('fontHeading')};
+  font-size: 14px;
+  line-height: 24px;
+  text-align: center;
+  a {
```

Comment:
Couldn't find a way to style specific styled components inside other styled components. Suggestions welcomed.

Reply:
I think these styles should be applied to the component passed in by *that* component.

Reply:
On one level this style rule makes sense: we specify the text colour for this element and we want all other child elements to have the same text colour. But using a naked `a` tag means we have to replicate any styling and behaviour attached to `Link`. What's more resend would probably be implemented using a `button` (via `Action`).

As discussed I think the best option is to create a separate `RibbonAction` component that applies the necessary text colour by extending `Action`.

Reply:
It seems that it's possible to apply style to specific styled components inside other styled components[1]. The problem is that the component we want to restyle is not actually a styled component. I will continue this discussion on coko's mattermost. For now I guess it's better to do the Action restyling in a different component so it's easier to change later on.

[1] - https://www.styled-components.com/docs/advanced#referring-to-other-components